voting/v2/helper/contracts.ts

```diff
@@ -0,0 +1,59 @@
+import { Voting } from './../typechain-types';
```

Comment:
@xiangdev1016 currently, we are using the default "../types" folder for most of the smart contract projects (reference: https://github.com/Hector-Network/hector-contracts/blob/main/bonding-farm/helper/contracts.ts). Please remove references to "/typechain-types" and use "./types" instead so that all projects are consistent.